### ISSUE TITLE
Find entry - button 

### ISSUE THREAD

**Issue by PolinaMandrik** [kartevonmorgen/kartevonmorgen.ts#72]
Is it possible to "teach" our search Box on the Main Page to see also the names of the added entries?

**Comment by PolinaMandrik** [comment]
![IMG_20211222_125241_739](https://user-images.githubusercontent.com/96134504/147073246-deecc468-8dc0-4c59-9261-2468fa7ca6e6.jpg)
![IMG_20211222_125236_331](https://user-images.githubusercontent.com/96134504/147073249-2fb34f80-85f1-4642-acd8-9c149a98c997.jpg)

**Comment by wellemut** [comment]
@PolinaMandrik the left means search for places and the right one to search for entries (with a global bounding box)?

What would happen next? Where can people enter their interest?

Could you please make the designs in English? That would be really helpful 🤠

**Comment by PolinaMandrik** [comment]
@wellemut yes, the left means: Search for an entry. Right Button is for: Add a New entry.
When you click with a cursor on of it, here come 3 buttons below: Project, Organization, Event to specify the type of entry.

**Comment by PolinaMandrik** [comment]
After you chose "Find a new entry", comes next line "What do we search for?" -here you can entry f.e. upsala circus, krug theatre etc
![IMG_20211223_141051_807](https://user-images.githubusercontent.com/96134504/147232248-ddbf1146-07a5-4d4a-8942-3aa46349c6b3.jpg)

**Comment by wellemut** [comment]
Wow that is quite a fancy button desing/ interaction.
So you never can use both fileds, (which is ok, maybe even good because less confusing...)
